src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistDialogForm.tsx

```diff
@@ -44,27 +45,48 @@ const MSG = defineMessages({
     id: 'dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.whitelisted',
     defaultMessage: 'Whitelisted',
   },
+  inputSuccess: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.inputSuccess`,
+    defaultMessage: `Address is whitelisted now. You can add another one or close modal.`,
+  },
+  fileSuccess: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.fileSuccess`,
+    defaultMessage: `File was added. You can add another one or close modal.`,
+  },
 });
 
 interface Props {
   back: () => void;
   colony: Colony;
+  showInput: boolean;
+  toggleShowInput: () => void;
+  formSuccess: boolean;
+  setFormSuccess?: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
 const ManageWhitelistDialogForm = ({
   back,
   colony,
   values,
+  errors,
+  isValid,
+  isSubmitting,
+  handleSubmit,
+  showInput,
+  toggleShowInput,
+  formSuccess,
+  setFormSuccess,
 }: Props & FormikProps<FormValues>) => {
   const { walletAddress, username, ethereal } = useLoggedInUser();
   const [tabIndex, setTabIndex] = useState<number>(0);
-
   const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
-
   const hasRegisteredProfile = !!username && !ethereal;
-
   const userHasPermission = hasRegisteredProfile && hasRoot(allUserRoles);
 
+  const { isWhitelistExtensionEnabled } = useEnabledExtensions({
+    colonyAddress: colony.colonyAddress,
+  });
```

Comment:
we dont need this

Reply:
@alicjakujawa no problem........ i'll simply remove the check.